src/phoenix/server/api/routers/evaluation_handler.py

```diff
@@ -100,11 +100,15 @@ async def _process_pyarrow(self, request: Request, project_name: str) -> Respons
         return Response(
             background=BackgroundTask(
                 self._add_evaluations,
+                request,
                 evaluations,
                 project_name,
             )
         )
 
-    async def _add_evaluations(self, evaluations: Evaluations, project_name: str) -> None:
+    async def _add_evaluations(
+        self, request: Request, evaluations: Evaluations, project_name: str
```

Comment:
nit. maybe use the State instead, so it's a bit more explicit
```suggestion
        self, state: State, evaluations: Evaluations, project_name: str
```

Reply:
Good idea

Reply:
addressed in ae16f08aa3e1222fef7d7bd379e5a7bde41b88ea